### ISSUE TITLE
enable cookies

### ISSUE THREAD

**Issue by bortkiewicz** [Ways-to-not-make-money/wtnmm-javascript#2]
Does this use cookies to store the current stats and stuff?

**Comment by hanyuone** [comment]
How?

**Comment by bortkiewicz** [comment]
Like it stores data somewhere (mainly on the user's computer) to store the data for later use?

**Comment by hanyuone** [comment]
No, I understand that - I mean, how are we going to use cookies? How are we going to store data in the user's computer?

**Comment by testitem** [comment]
Either actually write files:

```javascript

var f = new File("c:/game/savefile.txt");
f.open("w");
f.writeln("THE SAVEFILE DATA HERE");

// reading

var f = new File("c:/game/savefile.txt");
file.open("r"); // open file with read access
var str = "";
while (!file.eof) {
	// read each line of text
	str += file.readln() + "\n";
}
file.close();
alert(str);
```

I'm not sure about cookies.

**Comment by testitem** [comment]
ADAPTED FROM https://gist.github.com/Arahnoid/9925725

**Comment by testitem** [comment]
note: 
> Must be written in the node.js environment.

**Comment by bortkiewicz** [comment]
Forget that. Upload all of the data onto a repository on Github.

**Comment by bortkiewicz** [comment]
https://www.google.com/search?q=javascript+write+to+file&ie=utf-8&oe=utf-8#q=https+cookies

**Comment by bortkiewicz** [comment]
HELLO, @DerpfacePython ?

**Comment by hanyuone** [comment]
What? This issue is practically solved. Also, can you please style the page, @testitem? Thanks.

**Comment by bortkiewicz** [comment]
Is not solved! Oh, I styled the page already.

**Comment by hanyuone** [comment]
@testitem When I said styling - I meant style the actual game itself, not the page that holds the game. In fact, I don't think that extra bit at the top should even be there at all - this is meant to be a game, after all, not a webpage. Just fix up the styling of the actual game so that it looks better.

We can do cookies after we:
- style the game properly
- update the game to its current status
- find a way to auto-convert CoffeeScript in-browser (I.e. when the website is launched inside the browser).

**Comment by hanyuone** [comment]
Hello, are you reading this, @testitem?

**Comment by bortkiewicz** [comment]
Yes, of course I am? I am still working on more upgrades

**Comment by hanyuone** [comment]
@testitem Like what?

**Comment by bortkiewicz** [comment]
Pretty much all the upgrades in P0.7 and upgrade hiding

**Comment by hanyuone** [comment]
I'm still trying to figure out how to include cookies (using js-cookie, which I'll include in the `require` folder). Just an update.

**Comment by bortkiewicz** [comment]
It's easier when you read the entire Cookie Clicker source code because they use cookies to store data. Well, the source code is much too long for you to read it in a short amount of time.

**Comment by bortkiewicz** [comment]
I added js-cookie. Now i'm getting thoroughly annoyed that I have to import jc-cookie from main.js when they are in two different branched folders.

**Comment by hanyuone** [comment]
Yeah, js-cookie is great... but I just have no idea how it works.

**Comment by bortkiewicz** [comment]
Uh.. docs?

**Comment by bortkiewicz** [comment]
i.e. https://github.com/js-cookie/js-cookie/wiki?

**Comment by hanyuone** [comment]
I just have no idea how to access the cookie itself with `Cookie.get()`, do I declare it as a variable? Do I just call the function? What do I do?

**Comment by bortkiewicz** [comment]
well basically the readme animation says all.

Make a cookie: Cookie.set(name, value)
Get the data: Cookie.get(name)
Remove the cookie: Cookie.remove(name)

**Comment by bortkiewicz** [comment]
NO. No variable

**Comment by bortkiewicz** [comment]
just like that.

P.S. I need to eat breakfast now. Bye

**Comment by bortkiewicz** [comment]
Oh well just encode the data into a cookie.

**Comment by bortkiewicz** [comment]
i.e. `<money>|<AC-s>|<AC-price>|<MP-s>|<MP-price>|<SC-s>|<SC-price>|<CC-s>|<CC-price>|<BH-s>|<BH-price>|<the-upgrades-purchased>`
